client-tools/Shell/TelemetricsHandler.cpp

```diff
@@ -335,18 +337,15 @@ void TelemetricsHandler::getTelemetricsInfo(velocypack::Builder& builder) {
 
 void TelemetricsHandler::arrangeTelemetrics() {
   try {
+#ifndef ARANGODB_ENABLE_MAINTAINER_MODE
```

Comment:
Is it needed here? That makes check on line 348 a bit useless as it is always true in non-maintainer mode.

Reply:
It's not needed, but actually because the value is set to true already in the constructor for non-maintainer mode, but the function that fetches temeletrics restricts is to only being true if it's single server or it's a coordinator, so the value might have to change there
I removed it